### ISSUE TITLE
How to run test on Selenium Grid.

### ISSUE THREAD

**Issue by mandeephivehome** [klassijs/klassi-js#31]
Hi @larryg01 ,
I have setup Docker Selenium Grid, I can see from index page that remote driver details can be passed for browserstack as “-w, --remoteService [optional]”

Please can you advice what will be the best way to pass the grid URL as options.

Thanks

**Comment by mandeephivehome** [comment]
Hi @larryg01 ,
Thanks for your response,  I can run Selenium grid locally without passing and arguments. But my grid is not local. Let me rephrase my question. How can I pass the selenium grid URL when, grid is set up on an external server? 
http://XX.XXX.XXX.XX:4444/wd/hub
Thanks

**Comment by larryg01** [comment]
by starting the Grid at the same time as the test, think of it like starting browserstack from the command line
(i.e. selenium-grid start  && test start) 
and since the framework kills the process at the end of the test it will automatically close the grid. hope that helps.

**Comment by mandeephivehome** [comment]
I am sorry, but seems I am not able to make question clear and apologies for taking your time.

I can start a grid on my local machine and run the test. Driver is created for http://52.212.140.72:4444 and test is done.

But when I am running the test on an external server, then I need to pass the address for the http://xx.xxx.xxx.xx:4444/wd/hub for driver to run the test on server. I can run it by making the following changes in chromeDriver.js

    defaults = {
      host: http://xx.xxx.xxx.xx:4444/wd/hub, (NEW LINE ADDED)
      desiredCapabilities: {
        browserName: 'chrome',
        javascriptEnabled: true,
        acceptSslCerts: true,
        chromeOptions: {
          args: [
            // '--incognito',
            '--start-maximized'
          ]
        },
        path: chromedriver.path (REMOVE THIS LINE)
      }

I was looking, if this was already implemented. So I dont have to make the changes in code and pass the grid hub address through comment. I not then I can fork the repo and implement the functionality to run it for selenium grid.

Thanks

**Comment by larryg01** [comment]
forking the repo and implementing that functionality is good but have you thought about what you will have to do to run the test locally again.
It would be better if you made a switch and pass it as a parameter to the test that way you wont have to make code changes to switch from remote to local grid

**Comment by mandeephivehome** [comment]
Yeah, I will make it configurable.

Thanks for you time and help
